pkg/asset/manifests/operators.go

```diff
@@ -191,6 +191,7 @@ func (m *Manifests) generateBootKubeManifests(dependencies asset.Parents) []*ass
 		"app-version-kind.yaml":                      []byte(bootkube.AppVersionKind),
 		"app-version-tectonic-network.yaml":          []byte(bootkube.AppVersionTectonicNetwork),
 		"etcd-service.yaml":                          []byte(bootkube.EtcdServiceKubeSystem),
+		"kube-system-configmap-etcd-serving-ca.yaml": []byte(bootkube.KubeSystemConfigmapEtcdServingCA),
```

Comment:
Have you tested this?  template substitution appears to happen in the stanza above, though even so I think the parameters used don't match this pull.

Reply:
Fixed